### PR TITLE
Update font-logos with new linux icons

### PR DESCRIPTION
**Pull request by daniruiz**
#### Description

font-logos is outdated and now comes with some extra Linux-related logos. This PR contains the new font but I've not patched any font, in case I broke something. But I did test it with the fira-code font, especially to check the new kali-linux logo that I've recently added to font-logos in another  [PR](https://github.com/lukas-w/font-logos/pull/56)

#### Requirements / Checklist

- [x] Read the [Contributing Guidelines](https://github.com/ryanoasis/nerd-fonts/blob/master/contributing.md)
- [x] Read or at least glanced at the [FAQ](https://github.com/ryanoasis/nerd-fonts/wiki/FAQ-and-Troubleshooting)
- [x] Read or at least glanced at the [Wiki](https://github.com/ryanoasis/nerd-fonts/wiki)
- [x] Scripts execute without error (if necessary):
  - If any of the scripts were modified they have been tested and execute without error, e.g.:
    - `./font-patcher Inconsolata.otf --fontawesome --octicons --pomicons`
    - `./gotta-patch-em-all-font-patcher\!.sh Hermit`
- [x] Extended the README and documentation if necessary, e.g. You added a new font please update the table

#### What does this Pull Request (PR) do?
Adds new icons from [font-logos](https://github.com/lukas-w/font-logos)

#### How should this be manually tested?
I created a branch with only fira-code patched, in order to easily test it without rebuilding all the fonts
https://github.com/daniruiz/nerd-fonts/tree/fira%2Bkali/patched-fonts/FiraCode

#### Screenshots (if appropriate or helpful)
image

### CODE REVIEW

**Comment by daniruiz** [comment]
Any update or anything I can help?

**Comment by nourkagha** [comment]
Any updates on this? Would love to have the patched fonts updated with the Pop!_OS icon.

File: bin/scripts/lib/i_linux.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
-# Font Logos (Font Linux) (29 icons)
-# Codepoints: Nerd Fonts moved F100-F11C with holes → F300-F31C
+# Font Logos (Font Linux) (44 icons)
```
Comment (ryanoasis) [inline bin/scripts/lib/i_linux.sh:2]: Thank you for finding and updating the lib script as well!

File: font-patcher [hunk 2]
```diff
@@ -528,7 +528,7 @@ class font_patcher:
...
             {'Enabled': self.args.weather,              'Name': "Weather Icons",           'Filename': "weathericons-regular-webfont.ttf", 'Exact': False,                               'SymStart': 0xF000, 'SymEnd': 0xF0EB, 'SrcStart': 0xE300, 'SrcEnd': 0xE3EB, 'ScaleGlyph': None,             'Attributes': SYM_ATTR_DEFAULT},
-            {'Enabled': self.args.fontlinux,            'Name': "Font Logos (Font Linux)", 'Filename': "font-logos.ttf",                   'Exact': self.fontlinuxExactEncodingPosition, 'SymStart': 0xF100, 'SymEnd': 0xF11C, 'SrcStart': 0xF300, 'SrcEnd': 0xF31C, 'ScaleGlyph': None,             'Attributes': SYM_ATTR_DEFAULT},
+            {'Enabled': self.args.fontlinux,            'Name': "Font Logos (Font Linux)", 'Filename': "font-logos.ttf",                   'Exact': self.fontlinuxExactEncodingPosition, 'SymStart': 0xF100, 'SymEnd': 0xF12D, 'SrcStart': 0xF300, 'SrcEnd': 0xF32D, 'ScaleGlyph': None,             'Attributes': SYM_ATTR_DEFAULT},
```
Comment (ryanoasis) [inline font-patcher:531]: Seems good, no conflicts as of now :)

**Review by ryanoasis — APPROVED** [review]

**Comment by ryanoasis** [comment]
>  I've not patched any font, in case I broke something. But I did test it with the fira-code font

No worries, it should be okay. That should be reasonable enough

> I created a branch with only fira-code patched, in order to easily test it without rebuilding all the fonts

Thank you for doing a test, I also did a minimal test and it looks great.

> without rebuilding all the fonts

I am glad you did not do this and further it might be wasted effort depending on what else is to be changed, always best to shove as much as possible before rebuilding ALL the fonts 😄

**Comment by daniruiz** [comment]
Awesome! Thank you :wink:

**Pull request merged**